app/policies/policy_helpers.rb

```diff
@@ -9,11 +9,11 @@ def valid_coi_statement(user)
     statement.present? && !statement.expired?
   end
 
-  def belongs_to_acting_as_organization?(user)
-    if user.acting_as_organization_id.nil?
+  def belongs_to_action_organization?(user)
```

Comment:
should the be broken out maybe into two things `belongs_to_action_organization` and `has_no_organization`?

Reply:
Yeah, I'm not sure what the best way to handle this is, if the api request doesn't specify an organization then that's fine and the check should pass,  but you're right that its perhaps two conceptually different things